### ISSUE TITLE
Malformed .web_data.json resulting in: JSON.parse: expected property name or '}' at line 2 column 1 of the JSON data

### ISSUE THREAD

**Issue by h7io** [jlesage/docker-baseimage-gui#76]
Hi,

I have the following `Dockerfile`:
```Dockerfile
# Pull base image.
FROM jlesage/baseimage-gui:ubuntu-18.04-v4.0.3

RUN apt-get update && apt-get -y install libreoffice

# Copy the start script.
COPY startapp.sh /startapp.sh

# Set the name of the application.
RUN set-cont-env APP_NAME "libreoffice"
```

and the following `startapp.sh`:
```bash
#!/bin/sh
export HOME=/config

exec /usr/bin/libreoffice --calc
```

and when I open the URL to see the app I see:

![image](https://user-images.githubusercontent.com/1141177/195817613-dce81045-3675-4a08-ba90-19629ab5a593.png)

I have `docker exec`'d into the container and see this:
![image](https://user-images.githubusercontent.com/1141177/195817862-b9385eab-fe01-4144-b464-f899c9e2e92b.png)

I think the `-ne` strings should not be there.

Any idea what could the issue be?

Really liking the idea and docker image. Outstanding work.

**Comment by jlesage** [comment]
Thanks for reporting, I'm fixing this.

**Comment by jlesage** [comment]
Fixed in version 4.0.4.
